packages/flutter/lib/src/semantics/binding.dart

```diff
@@ -125,6 +142,13 @@ mixin SemanticsBinding on BindingBase {
         arguments is ByteData
             ? action.copyWith(arguments: const StandardMessageCodec().decodeMessage(arguments))
             : action;
+    final List<ValueSetter<ui.SemanticsActionEvent>> localListeners = _semanticsActionListeners
```

Comment:
It's not immediately obvious what this is trying to do. AFAICT, this is creating a defensive copy to make sure the iterated list is not mutated during the loop. Maybe leave a comment, or give the variable a better name e.g. `defensiveCopy` or something?

Reply:
I'll leave a comment. This is a pattern that we use everywhere in the framework where listeners need to be called, though.